zebra/zebra_srv6_vty.c

```diff
@@ -938,6 +940,40 @@ DEFPY (locator_behavior,
 	return CMD_SUCCESS;
 }
 
+DEFPY (locator_flavor_psp,
+       locator_flavor_psp_cmd,
+       "[no] flavor psp",
+       NO_STR
+       "Configure SRv6 flavors\n"
+       "Specify Penultimate Segment Popping flavor\n")
+{
+	VTY_DECLVAR_CONTEXT(srv6_locator, locator);
+
+	if (no && !CHECK_FLAG(locator->flags, SRV6_LOCATOR_PSP))
+		/* SRv6 locator PSP flag already unset, nothing to do */
+		return CMD_SUCCESS;
+
+	if (!no && CHECK_FLAG(locator->flags, SRV6_LOCATOR_PSP))
+		/* SRv6 locator PSP flag already set, nothing to do */
+		return CMD_SUCCESS;
+
+	if (!locator->sid_format)
+		/* Remove old locator from zclients */
+		zebra_notify_srv6_locator_delete(locator);
+
+	/* Set/Unset the SRV6_LOCATOR_USID */
+	if (no)
+		UNSET_FLAG(locator->flags, SRV6_LOCATOR_PSP);
+	else
+		SET_FLAG(locator->flags, SRV6_LOCATOR_PSP);
+
+	if (!locator->sid_format)
```

Comment:
Guard `if (!locator->sid_format)` should be removed.

Reply:
done